src/app/components/cds/media-cards/Card.module.css

```diff
@@ -53,11 +53,12 @@
   background-color: var(--color-white-100);
   border: 1px solid var(--color-gray-88);
   border-radius: 8px;
-  padding: 16px;
+  padding: 8px;
   width: 100%;
 
   &:hover {
     border-color: var(--color-blue-32);
+    outline: solid 1px var(--color-blue-32);
```

Comment:
I am adding an outline to the hover state in order to get a slightly thicker border on hover but not changing the page layout because of the added border